lib/gringotts/gateways/sagepay.ex

```diff
@@ -40,6 +21,23 @@ defmodule Gringotts.Gateways.SagePay do
   [home]: sagepay.co.uk
   [docs]: integrations.sagepay.co.uk
 
+  ## The `opts` argument
+
+  Most `Gringotts` API calls accept an optional `keyword` list `opts` to supply
+  [optional arguments][extra-arg-docs] for transactions with the MONEI
+  gateway. The following keys are supported:
+
+  | Key                      |
+  |  ---------------         |
+  |  [`merchant_id`]         |
+  |  [`vendor`]              |
+  |  [`vendorTxcode`]        |  
```

Comment:
1. Please follow the elixir style for these atoms (snake_case).
2. Why are these surrounded by `[]`?